src/lxc/start.h

```diff
@@ -35,6 +35,7 @@
 #include "namespace.h"
 
 struct lxc_handler {
+	bool root;
```

Comment:
Out of context, 'root' could mean many things.  How about sticking with am_root?